matpower/matpower-converter/src/main/java/com/powsybl/matpower/converter/MatpowerImporter.java

```diff
@@ -0,0 +1,469 @@
+            voltageLevel = substation.newVoltageLevel()
+                    .setId(voltageLevelId)
+                    .setNominalV(nominalV)
+                    .setTopologyKind(TopologyKind.BUS_BREAKER)
+                    .add();
+        }
+        return voltageLevel;
+    }
+
+    private static void createLoad(MBus mBus, VoltageLevel voltageLevel) {
+        if (mBus.getRealPowerDemand() != 0 || mBus.getReactivePowerDemand() != 0) {
+            String busId = getBusId(mBus.getNumber());
+            String loadId = busId + "-L";
+            LOGGER.debug("Creating load {}", loadId);
+            voltageLevel.newLoad()
+                .setId(loadId)
+                .setConnectableBus(busId)
+                .setBus(busId)
+                .setP0(mBus.getRealPowerDemand())
+                .setQ0(mBus.getReactivePowerDemand())
+                .add();
+        }
+    }
+
+    private static String createGenId(MGen mGen, Network network) {
+        String genIdPrefix = "G" + mGen.getNumber() + "-";
+        int uniqueGenSuffix = 0;
+        String genId;
+        do {
+            genId = genIdPrefix + uniqueGenSuffix++;
+        } while  (network.getIdentifiable(genId) != null);
+        return genId;
+    }
+
+    private static void createShuntCompensator(MBus mBus, VoltageLevel voltageLevel, PerUnitContext perUnitContext) {
+        if (mBus.getShuntSusceptance() != 0) {
+            String busId = getBusId(mBus.getNumber());
+            String shuntId = busId + "-SH";
+            LOGGER.debug("Creating shunt {}", shuntId);
+            double zb = Math.pow(voltageLevel.getNominalV(), 2) / perUnitContext.getBaseMva();
+            voltageLevel.newShuntCompensator()
+                    .setId(shuntId)
+                    .setConnectableBus(busId)
+                    .setBus(busId)
+                    .setCurrentSectionCount(1)
+                    .newLinearModel()
+                        .setMaximumSectionCount(1)
+                        .setbPerSection(mBus.getShuntSusceptance() / perUnitContext.getBaseMva() / zb)
+                        .add()
+                    .add();
+        }
+    }
+
+    private static String getBranchId(char type, int from, int to, Network network) {
+        String id;
+        int uniqueCircuit = 1;
+        do {
+            id = "" + type + from + "-" + to + "-" + uniqueCircuit++;
+        } while (network.getIdentifiable(id) != null);
+        return id;
+    }
+
+    private static boolean isInService(MBranch branch) {
+        return Math.abs(branch.getStatus()) > 0;
+    }
+
+    private static boolean isInService(MGen generator) {
+        return generator.getStatus() > 0;
+    }
+
+    private static void createLine(MBranch branch, ContainersMapping containerMapping, Network network, PerUnitContext perUnitContext) {
+        String lineId = getBranchId('L', branch.getFrom(), branch.getTo(), network);
+        LOGGER.debug("Creating line {}", lineId);
```

Comment:
To be consistent, maybe you should also log bus1 and bus2 like for transformers

Reply:
done.

Reply:
fixed: used a prefix, consistently